angular/modules/http/README-ENG.md

```diff
@@ -1,10 +1,10 @@
 # [Angular schedule.](../../README.md) Http
 
-## In this module, students need to learn:
+## In this module, you will learn:
 
-- `HttpClientModule` Module.
-- `HttpTypeEvent` Object.
-- Intercept of HTTP requests with `HTTP Interceptor`.
+- How to use the `HttpClientModule` in Angular
```

Comment:
`HttpClientModule` is deprecated

Reply:
agree. it is worth mentioning the `provideHttpClient()`. but we can keep them alongside with a note that the `HttpClientModule` is suppressed in favor of the new standalone approach